tests/kafkatest/tests/kafka_test.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, test_context, num_zk, num_brokers, topics=None):
         self.kafka = KafkaService(
             test_context, self.num_brokers,
             self.zk, topics=self.topics,
-            controller_num_nodes_override=self.num_zk)
+            controller_num_nodes_override=self.num_zk,
+            dynamicRaftQuorum=True)
```

Comment:
This is needed for now for getting the tests to pass, there will be a follow-up PR to remove

Reply:
removed now